src/Core/src/Fonts/FontManager.Android.cs

```diff
@@ -35,25 +35,6 @@ public float GetScaledPixel(Font font)
 			if (font.IsDefault)
 				return 14;
 
-			if (font.UseNamedSize)
```

Comment:
Is this going to be implemented again at the controls level?

Reply:
Restored.